app/scripts/OverlayTrack.js

```diff
@@ -234,6 +322,8 @@ class OverlayTrack extends PixiTrack {
       height = minHeight;
     }
 
+    options.isVertical = true;
```

Comment:
Not sure why this is here but I presume it's intentional?

Reply:
Yes it is. it's part of the function to draw a vertical annotation